### PR TITLE
Feat/2236012071 tabs atomic component

### PR DESCRIPTION
**Pull request by rick23p**
https://sovryn.monday.com/boards/2218344956/pulses/2236012071

### CODE REVIEW

File: src/app/components/Tabs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  initial,
+  onChange,
+  className,
+  contentClassName,
+}) => {
+  const [activeTab, setActiveTab] = useState(initial);
+
+  const selectTab = (item: ITabItem) => {
+    if (item.disabled) return;
+    setActiveTab(item.id);
```
Comment (tiltom) [inline src/app/components/Tabs/index.tsx:30]:
```suggestion
    if (item.disabled) {
      return;
    }
    setActiveTab(item.id);
```

File: src/app/components/Tabs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+export const Tabs: React.FC<TabsProps> = ({
+  items,
+  initial,
+  onChange,
+  className,
+  contentClassName,
+}) => {
+  const [activeTab, setActiveTab] = useState(initial);
+
+  const selectTab = (item: ITabItem) => {
```
Comment (tiltom) [inline src/app/components/Tabs/index.tsx:28]: Use `useCallback`.

**Review by tiltom — CHANGES_REQUESTED** [review]

**Review by tiltom — DISMISSED** [review]

**Comment by pietro-maximoff** [comment]
just a quick suggestion here, maybe make sense to add an option that will allow you to combine all tabs into one select element based on media breakpoints, its more useful for mobile devices.
for instance if we have 3 or more tabs then we can hide them on mobile and show one select element, that's will be more conveniently.
Screenshot at Mar 10 23-22-01

**Review by JoeStorm99 — APPROVED** [review]

**Comment by ShynRou** [comment]
@pietro-maximoff That's a good point, but I think that should be something we address at a later stage.

File: src/app/components/Tab/index.tsx [hunk 3]
```diff
@@ -1,41 +1,33 @@
...
+  disabled,
+  className,
+}) => (
+  <span
+    className={classNames(
+      'tw-text-sov-white tw-px-2.5 tw-py-2 tw-text-lg tw-font-thin tw-whitespace-nowrap',
+      {
+        'tw-text-white tw-font-normal': active,
+        'tw-cursor-pointer hover:tw-text-gray-9': !disabled,
+        'tw-opacity-50': disabled,
```
Comment (ShynRou) [inline src/app/components/Tab/index.tsx:25]:
```suggestion
        'tw-opacity-50 tw-cursor-not-allowed': disabled,
```

File: src/app/components/Tabs/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    <div className={className}>
+      <div className="tw-flex tw-items-center tw-flex-nowrap tw-overflow-auto">
+        {items.map(item => (
+          <Tab
+            key={item.id}
+            active={item.id === activeTab}
+            disabled={item.disabled}
+            onClick={() => selectTab(item)}
+            text={item.label}
+            className="tw-mr-2"
```
Comment (ShynRou) [inline src/app/components/Tabs/index.tsx:54]:
```suggestion
            className="tw-mx-2"
```

File: src/app/components/Tab/index.tsx [hunk 5]
```diff
@@ -1,41 +1,33 @@
...
-    `}
-`;
+export const Tab: React.FC<TabProps> = ({
+  text,
+  active,
+  onClick,
+  disabled,
+  className,
+}) => (
+  <span
```
Comment (ShynRou) [inline src/app/components/Tab/index.tsx:19]: Can you please make this into a button, so it is focus-able and shows up on screen readers.

**Review by ShynRou — CHANGES_REQUESTED** [review]

**Review by soulBit — DISMISSED** [review]
Reviewed code and resolved previous comments

**Pull request merged**